release_notes/369-release-notes.adoc

```diff
@@ -83,8 +83,12 @@ In {product-title} 3.69, when Collector detects that the host is using UEFI secu
 === Scanner memory limit increases
 Red Hat has increased the default Scanner memory limit from `3000` MiB to `4` GiB.
 
+[id="known-issues-369"]
+== Known issues
+* *ROX-9750*: The `FROM` instruction in the `DISALLOWED DOCKERFILE LINE` policy field is not recognized by {product-title-short}. For example, creating a policy that disallows `FROM:unwanted.example.com` in the Dockerfile will not generate a policy violation. 
```

Comment:
```suggestion
* *ROX-9750*: The `FROM` instruction in the `DISALLOWED DOCKERFILE LINE` policy field is not recognized by {product-title-short}. For example, creating a policy that disallows `FROM:unwanted.example.com` in the Dockerfile does not generate a policy violation. 
```
[ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) recommends avoiding future tense.

Reply:
Fixed - thank you!